modules/tinymce/src/plugins/template/test/ts/browser/DialogGetPreviewContentTest.ts

```diff
@@ -56,58 +60,56 @@ const corsStyleAndContent = '<!DOCTYPE html><html><head>' +
-  TemplatePlugin();
-  SilverTheme();
-
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const tinyApis = TinyApis(editor);
-
-    const sCheckPreview = (expected: string, html?: string) =>
-      Step.sync(() =>
-        Assert.eq('', expected, getPreviewContent(editor, html ? html : '')));
-
-    Pipeline.async({}, [
-      Log.stepsAsStep('TINY-6115', 'Dialog.getPreviewcontent: No CORS or content style, no previous HTML', [
-        sCheckPreview(noCorsNoStyle)
-      ]),
-      Log.stepsAsStep('TINY-6115', 'Dialog.getPreviewcontent: CORS but no content style, no previous HTML', [
-        tinyApis.sSetSetting('content_css_cors', true),
-        sCheckPreview(corsNoStyle),
-        tinyApis.sDeleteSetting('content_css_cors')
-      ]),
-      Log.stepsAsStep('TINY-6115', 'Dialog.getPreviewcontent: No CORS but content style, no previous HTML', [
-        tinyApis.sSetSetting('content_style', 'This is the style inserted into the document'),
-        sCheckPreview(noCorsStyle),
-        tinyApis.sDeleteSetting('content_style')
-      ]),
-      Log.stepsAsStep('TINY-6115', 'Dialog.getPreviewcontent: CORS and content style, no previous HTML', [
-        tinyApis.sSetSetting('content_style', 'This is the style inserted into the document'),
-        tinyApis.sSetSetting('content_css_cors', true),
-        sCheckPreview(corsStyle),
-        tinyApis.sDeleteSetting('content_style'),
-        tinyApis.sDeleteSetting('content_css_cors')
-      ]),
-      Log.stepsAsStep('TINY-6115', 'Dialog.getPreviewcontent: with provided content', [
-        tinyApis.sSetSetting('content_style', 'This is the style inserted into the document'),
-        tinyApis.sSetSetting('content_css_cors', true),
-        sCheckPreview(corsStyleAndContent, 'Custom content which was provided'),
-        tinyApis.sDeleteSetting('content_style'),
-        tinyApis.sDeleteSetting('content_css_cors')
-      ]),
-      Log.stepsAsStep('TINY-6115', 'Dialog.getPreviewcontent: with provided html', [
-        tinyApis.sSetSetting('content_style', 'This is the style inserted into the document'),
-        tinyApis.sSetSetting('content_css_cors', true),
-        sCheckPreview('<html>Custom content here', '<html>Custom content here'),
-        tinyApis.sDeleteSetting('content_style'),
-        tinyApis.sDeleteSetting('content_css_cors')
-      ])
-    ], onSuccess, onFailure);
-  }, {
-    theme: 'silver',
+describe('browser.tinymce.plugins.template.Dialog.getPreviewContent', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
     plugins: 'template',
-    toolbar: 'template',
-    indent: false,
     base_url: '/project/tinymce/js/tinymce'
-  }, success, failure);
+  }, [ Plugin, Theme ]);
+
+  const checkPreview = (expected: string, html: string = '') => {
+    const editor = hook.editor();
+    assert.equal(expected, getPreviewContent(editor, html));
+  };
+
+  const { addSettings, delSettings } = Settings(hook);
+
+  afterEach(delSettings);
```

Comment:
```suggestion
  afterEach(() => {
    delSettings()
  });
```
This is one of the cases where we don't want to ETA reduce something, because it's not safe to do so due to the return type being important for `afterEach`, among other things.